cloudformationTemplates/startChatContactAPI/customBuildIndex.html

```diff
@@ -99,7 +96,9 @@ <h1>Amazon Connect - Custom Implementation of Customer Chat</h1>
                             DisplayName: customerName
                         },
                         ContactFlowId: contactFlowId,
-                        InstanceId: instanceId
+                        InstanceId: instanceId,
+                        // Enable rich messaging support: https://docs.aws.amazon.com/connect/latest/adminguide/enable-text-formatting-chat.html
+                        SupportedMessagingContentTypes: ["text/plain", "text/markdown"] // include 'text/markdown'
```

Comment:
Why is this a list here but a string in other index.html files?

Reply:
`connect.startChatContact()` and ChatInitializer will convert the string to the array, before sending to APIGW. This file calls APIGW directly, without going through `amazon-connect-chat-interface.js`